packages/components/NEW-COMPONENT-CHECKLIST.md

```diff
@@ -5,13 +5,58 @@ This is the quest issue/engineering checklist for the COMPONENT_NAME Component.
+
+### Component Creation
+- [ ] [create a branch](https://help.figma.com/hc/en-us/articles/360063144053-Create-branches-and-merge-changes#Create_a_branch) in the [product components ui kit](https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/HDS-Product---Components)
+- [ ] add a new page for the component, as needed and mark the header as In progress
+- [ ] **design**
+  - [ ] work through design explorations, providing 2-3 visual directions based on existing [foundations](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?node-id=2916%3A4) and patterns, as well as the findings from the audit and requirements from the CRD
+  - [ ] review with appropriate stakeholders (typically HDS designers, engineering partner, ambassadors) and iterate, as necessary
+  - [ ] review for a11y considerations and iterate, as necessary
+  - [ ] review with design leadership and iterate, as necessary
+  - [ ] once approved, build out the design as a component
+    - ensure the variant and property names align with what was discussed/decided on with your engineering partner
+  - [ ] test the component with HDS designers
+- [ ] **documentation**
+  - [ ] stickersheet
+    - make sure the component instances can be easily selected and other layers are locked
+  - [ ] design guidelines
+    - when complete, lock this frame
+  - [ ] figma tips, as necessary
+    - when complete, lock this frame
+  - [ ] migration details, as necessary
+    - when complete, lock this frame
+- [ ] **release**
+  - lock and hide the components leaving only the stickersheets and docs visible, and change the header's badge to Experimental
+  - add release notes to the changelog
+  - [ ] review with situationally appropriate reviewers to get final approval
+  - [ ] merge the branch
+  - [ ] publish the ui kit
+  - do not communicate this release to consumers
+- [ ] **testing**
+  - [ ] add a new page to the [playground](https://www.figma.com/file/95YqaJF4TNV72ucXbyC7A0/Playground?node-id=302%3A10) file and prepare a frame for each ambassador to test
+  - [ ] test the component with ambassadors
+  - if iterations are needed, work through the process until approved
+
+When ready to add docs to the scrappy site:
+- [ ] export a screenshot of the design guidelines
+- [ ] create a new issue in [github](https://github.com/hashicorp/design-system/issues) with the screenshot
+- [ ] let your engineering partner know it's ready to be added (or add it yourself)
+
+
 ## Engineering Checklist
 The engineering checklist has six parts: creating the feature branch, component template, component backing class, component style, tests, and documentation.
 
+### Dependency
+- [Figma Design](url_here): Since we are trying to align the component API naming with the same terms used in the Figma file, it is likely useful to have a fairly stable Figma design before we create a component; it should definitely be finalized before the component ships, however.
```

Comment:
@heatherlarsen I don't think we need to add a link here, this list is not a GitHub template

```suggestion
- Since we are trying to align the component API naming with the same terms used in the Figma file, it is likely useful to have a fairly stable Figma design before we create a component; it should definitely be finalized before the component ships, however.
```